### DISCUSSION TITLE
MAX FOR LIVE device

### DISCUSSION THREAD

**Discussion by zwerden** [vcync/modV#831] [Ideas]
an official MODV Max for Live device could be a cool feature... I believe they are both in JS (MODV & M4L), and it could make for easier setup with less audio routing.

**Comment by 2xAA** [comment]
@zwerden Nice idea! Unfortunately I don't know anything about M4L - however if you've got some reading or API docs I'd happily take a look :)

**Comment by zwerden** [comment]
https://docs.cycling74.com/max7/vignettes/live_api_overview
https://docs.cycling74.com/max7/tutorials/javascriptchapter01#The_Global_Code
https://docs.cycling74.com/max7/vignettes/javascript_usage_topic
https://docs.cycling74.com/max7/vignettes/jsliveapi

HI thank You for a reply. Max for Live IS a paid software (included with Ableton Suite), so I'd understand if it doesn't align with You...also I think there may even be enough resources floating around online (if not a similar M4L device already) for someone to integrate MODV and LIVE... anyways, Thank You! cheers.
